waterArduino.INO

```diff
@@ -3,12 +3,20 @@ Servo waterServo;  //creates the name of your servo
 int sensPin = 0;  //assignes a pin for the soil sensor
 int servoPin = 3; //sets the servo pin, this must be a PWM pin
 int powerPin = 2; //assignes a pin to power the sensor. optional
-int wetVal = 500; //the sensor value where the soil is sufficiently wet
-int dryVal = 300; //the value where the soil is dry and needs watering
+//int wetVal = 500; //the sensor value where the soil is sufficiently wet
+//int dryVal = 300; //the value where the soil is dry and needs watering
 int dirtVal;    //variable to store one sensor reading
 int waterPosition = 170;
 int waterHold = 10;
 double avgDirt; //variable to store average of 10 sensor readings
+//long loopDelay = 300000; // 5 minutes in milliseconds
+//int numReads = 10;
+
+// below are test values to make debugging easier / faster:
+long loopDelay = 3000; // 3 seconds in milliseconds
+int numReads = 1;
```

Comment:
I created numReads because we use the value in the for loop and in calculating the average.  This allows us to change the number of loops without having to change the average calculation as well.